packages/cli/src/Command/Console.php

```diff
@@ -0,0 +1,225 @@
+
+namespace T3Docs\Cli\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class Console extends Command
+{
+    /**
+     * Maps a Settings.cfg key for [html_theme_options] to the XML <extension> element
+     */
+    private const MAPPING_SETTING = [
+        'project_home' => 'project-home',
+        'project_contact' => 'project-contact',
+        'project_repository' => 'project-repository',
+        'project_issues' => 'project-issues',
+        'project_discussions' => 'project-discussions',
+
+        'use_opensearch' => 'use-opensearch',
+
+        'github_revision_msg' => 'github-revision-msg',
+        'github_branch' => 'edit-on-github-branch',
+        'github_repository' => 'edit-on-github',
+        'github_sphinx_locale' => 'github-sphinx-locale',
+        'github_commit_hash' => 'github-commit-hash'
+    ];
+
+    /**
+     * Maps a Settings.cfg key for [general] to the XML <project> element
+     */
+    private const MAPPING_PROJECT = [
+        'project' => 'project',
+        'release' => 'release',
+        'version' => 'version',
+        'copyright' => 'copyright',
+    ];
+
+    /**
+     * Maps all Settings.cfg sections that are not covered by this converter
+     */
+    private const MAPPING_DEPRECATED_SECTIONS = [
+        'notify',
+        'latex_elements'
+    ];
+
+    /**
+     * Maps all Settings.cfg sections that are converted
+     */
+    private const MAPPING_ACCEPTED_SECTIONS = [
+        'html_theme_options',
+        'general',
+        'intersphinx_mapping'
+    ];
+
+    public function __construct(
+    ) {
+        parent::__construct('console');
+
+        $this->addArgument(
+            'input',
+            InputArgument::REQUIRED,
+            'Path to directory where settings.cfg is stored, and guides.xml will be generated.',
+        );
+
+        $this->addOption(
+            'force',
+            null,
+            InputOption::VALUE_NONE,
+            'When set, forces to overwrite possibly existing XML.'
+        );
+
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $settingsFile = $input->getArgument('input') . '/Settings.cfg';
+        $guidesFile = $input->getArgument('input') . '/guides.xml';
+
+        if (!file_exists($settingsFile)) {
+            $output->writeln('<error>Could not open ' . $settingsFile . '</error>');
+            return Command::FAILURE;
+        }
+
+        if (file_exists($guidesFile) && !$input->getOption('force')) {
+            $output->writeln('<error>Target file already exists in specified directory (' . $guidesFile . ')</error>');
+            return Command::FAILURE;
+        }
+
+        $output->writeln('Converting ' . $settingsFile . ' to ' . $guidesFile . ' ...');
```

Comment:
```suggestion
        $output->writeln('Migrating ' . $settingsFile . ' to ' . $guidesFile . ' ...');
```

should be a better word.